data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/ComponentPosition.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "ComponentPosition",
+  "doc": "The group index and the leaf index of the transaction component",
+  "namespace": "net.corda.data.ledger.persistence",
+  "fields": [
+    {
+      "name": "groupIndex",
```

Comment:
Do we need the group index here? Shouldn't that always be OutputStates?

Reply:
I was not aware that is always fixed, I'll change it

Reply:
fixed